cpp/include/utilities/cython.hpp

```diff
@@ -201,5 +199,8 @@ std::pair<size_t, weight_t> call_louvain(raft::handle_t const& handle,
                                          size_t max_level,
                                          weight_t resolution);
 
+// Helper for setting up subcommunicators
+void init_subcomms(raft::handle_t& handle, size_t row_comm_size);
```

Comment:
Public API header right? 
Users will wonder what `row_comm_size` is, how it should be set. Any specific default behavior should be documented as well.

Reply:
Thanks, working on that now for the next commit.